.github/workflows/pull-request-workflow.yml

```diff
@@ -33,7 +37,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16.x, 18.x]
```

Comment:
I think linting commit doesn't relate to nodejs version, so we can skip it

Reply:
Ok